### DISCUSSION TITLE
Anyone successfully set up TypeScript types with the official i18next way of doing it?

### DISCUSSION THREAD

**Discussion by brandonpittman** [sergiodxa/remix-i18next#110] [Q&A]
Got nowhere with [the i18next TypeScript setup](https://www.i18next.com/overview/typescript).

Anyone have this working? I saw [this gist](https://github.com/sergiodxa/remix-i18next/discussions/96) but that feels like a lot of extra and likely unnecessary work.

**Comment by Cmoen11** [comment]
Have you been able to figure this out?

**Reply by Cmoen11** [reply]
I've come up with a workaround https://github.com/sergiodxa/remix-i18next/discussions/179
